onyx-remote-driver/src/com/onyx/persistence/manager/impl/DefaultSocketPersistenceManager.java

```diff
@@ -89,7 +87,7 @@ public void saveEntities(List<? extends IManagedEntity> entities) throws EntityE
         try {
             proxyPersistenceManager.saveEntities(entities);
         } catch (RemoteException e) {
-            throw new EntityException(e);
```

Comment:
There is no need because, I shouldn't have been using the constructor to instantiate a new EntityException anyway because we had the instance of the root problem all along.